Classes/Controller/Backend/ControlCenter/Update/StepUpdateController.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Tvp\TemplaVoilaPlus\Controller\Backend\Update;
+namespace Tvp\TemplaVoilaPlus\Controller\Backend\ControlCenter\Update;
```

Comment:
This controller will leave us, after solving #372

Reply:
I know, but as long as it exists, it needs to be resolvable.